core/base/src/TApplication.cxx

```diff
@@ -1073,6 +1085,68 @@ void TApplication::Help(const char *line)
       }
       // We strip the command line after removing ".help" or ".?".
       strippedCommand = strippedCommand.Strip(TString::kBoth);
+      
+      if (strippedCommand == "edit") {
+         Printf("\nROOT terminal keyboard shortcuts (GNU-readline style).");
+         #ifdef R__MACOSX
+         #define FOOTNOTE " *"
+         Printf("* This command might be intercepted by macOS predefined system shortcuts.")
+         // https://apple.stackexchange.com/questions/18043/how-can-i-make-ctrlright-left-arrow-stop-changing-desktops-in-lion
+         #else
+         #define FOOTNOTE ""
+         #endif
+         Printf("==============================================================================");
+         Printf("   Alt+Backspace       : delete previous word" FOOTNOTE);
+         Printf("   Arrow Down          : navigate downwards in command history");
+         Printf("   Arrow Left          : move cursor left");
+         Printf("   Arrow Right         : move cursor right");
+         Printf("   Arrow Up            : navigate upwards in command history");
+         Printf("   Backspace           : remove previous character");
+         Printf("   CTRL+A              : move cursor to beginning of line");
+         Printf("   CTRL+Arrow Left     : jump to previous word" FOOTNOTE);
+         Printf("   CTRL+Arrow Right    : jump to next word" FOOTNOTE);
+         Printf("   CTRL+B              : move cursor left");
```

Comment:
Do we need to document that as the Left Arrow Dows the same ? or may be document it with left arrow: 
" Arrow Left  or CTRL+B: move cursor left"